src/System.Net.Http/src/System/Net/Http/HttpResponseMessage.cs

```diff
@@ -36,6 +36,8 @@ public Version Version
             }
         }
 
+        internal void SetVersionUnchecked(Version value) => _version = value;
```

Comment:
The equivalent API for headers is called "TryAddWithoutValidation".  Not sure this is better than just "Unchecked", but maybe we should be consistent.

Reply:
Ok, will rename.